runtime/autoload/man.vim

```diff
@@ -75,7 +75,13 @@ function man#get_page(...) abort
 
   setlocal modifiable
   silent keepjumps norm! 1G"_dG
-  let $MANWIDTH = winwidth(0)
+  if empty($MANWIDTH)
+    if &textwidth
+      let $MANWIDTH = &textwidth
+    else
+      let $MANWIDTH = winwidth(0)
+    endif
```

Comment:
Changed. I've forced pushed this as I'm not sure how else to alter an open pull request. Apologies if this is bad form.